### ISSUE TITLE
Hyperparameters of several finetuning experiments

### ISSUE THREAD

**Issue by wwsource** [princeton-vl/RAFT-Stereo#83]
Thank you for your great work!

Could you give the hyperparameters (like **learning rate**, **batch size**, **crop size**, and  **step number**) that can reproduce the paper data in three finetuning experiments (**KITTI**, **ETH3D** and **Middlebury**), which are not explicitly described in the paper "RAFT-Stereo: Multilevel Recurrent Field Transforms for Stereo Matching".

Thank you very much.

**Comment by lahavlipson** [comment]
These are the settings:

Middlebury: lr=2e-5,  batch_size 2, crop_size=384x1000, steps=4k
ETH3D: lr=1e-5,  batch_size 2, crop_size=384x1000, steps=4k saturation_range=[0 0] img_gamma=[0.5 2.0 0.8 1.2]
KITTI: lr=1e-5,  batch_size 3, crop_size=320x1000, steps=5k

**Comment by rmadan2** [comment]
Thank you for your response above.
Can you also please specify the number of disparity field updates configured for fine-tuning and inference, for all three datasets? Thank you.
